src/main/java/de/qabel/desktop/config/DefaultClientConfiguration.java

```diff
@@ -6,18 +6,28 @@
 import javafx.collections.ListChangeListener;
 import javafx.collections.ObservableList;
 
+import java.util.Date;
+import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.Observable;
 
 public class DefaultClientConfiguration extends Observable implements ClientConfiguration {
 	private Account account;
 	private Identity identity;
+	private HashMap<String, Date> lastDropMap = new HashMap<>();
+
 	private ObservableList<BoxSyncConfig> boxSyncConfigs = FXCollections.synchronizedObservableList(FXCollections.observableList(new LinkedList<>()));
 	private String deviceId;
 
 	public DefaultClientConfiguration() {
-		boxSyncConfigs.addListener((ListChangeListener) c -> {observeBoxSyncConfigs(); boxSyncConfigWasChanged(); });
+		boxSyncConfigs.addListener((ListChangeListener) c -> {
+			observeBoxSyncConfigs();
+			boxSyncConfigWasChanged();
+		});
 		observeBoxSyncConfigs();
+		if (lastDropMap == null) {
+			lastDropMap = new HashMap<>();
```

Comment:
This looks like dead code.
When lastDropMap was defined on declaration, it can't be `null` inside/after the constructor.
The only case lastDropMap could be null is when setLastDropMap is called, so the null-check should be either in setLastDropMap or it's callers (like ClientConfigurationRepositories).